src/Expression.php

```diff
@@ -614,13 +614,12 @@ public function execute($connection = null)
 
             try {
                 $statement->execute();
-            } catch (\Exception $e) {
-                $new = new Exception([
+            } catch (\PDOException $e) {
+                $new = new ExecuteException([
                     'DSQL got Exception when executing this query',
-                    'error' => $e->getMessage(),
+                    'error' => $e->errorInfo[2],
                     'query' => $this->getDebugQuery(),
-                ]);
-                $new->by_exception = $e;
+                ], $e->errorInfo[1]);
```

Comment:
@DarkSide666 I was not able to find any test testing that the SQL query/execution may fail.

Can you please add 1 test quering `SELECT non_existing_column` which should throw the new `ExecuteException` with `1054` code (`Unknown column` code) at the appropriate place? Thanks.